main function missing

Just dropping my stack trace here when I run the following command.

```sh
$ cargo mobile update
> ...
> ...
> ...
> error[E0601]: `main` function not found in crate `cargo_mobile`
>    --> src/bin/cargo-mobile.rs:223:2
>     |
> 223 | }
>     |  ^ consider adding a `main` function to `src/bin/cargo-mobile.rs`
> For more information about this error, try `rustc --explain E0601`.
> error: could not compile `tauri-mobile` due to previous error
> error: failed to compile `tauri-mobile v0.2.0 (/Users/<username>/.tauri-mobile/checkouts/tauri-mobile)`, intermediate artifacts can be found at `/Users/<username>/.tauri-mobile/checkouts/tauri-mobile/target`
> error: Failed to update `tauri-mobile`
>     Failed to install new version of `tauri-mobile`: Command "cargo install --force --path /Users/<username>/.tauri-mobile/checkouts/tauri-mobile --no-default-features --features " didn't complete
>     successfully, exiting with code 101.
```

I'm using an ARM-based MacBook Pro M1. And Rust 1.65.0

```sh
$ rustc --version                                                                                                                                                                       
> rustc 1.65.0 (897e37553 2022-11-02)
```

--- a/.changes/fix-cargo-mobile-update.md
+++ b/.changes/fix-cargo-mobile-update.md
@@ -1,0 +1,5 @@
+---
+'tauri-mobile': patch
+---
+
+Fix `cargo mobile update` target branch and enabled `cli` feature when update.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -16,6 +16,10 @@ categories = [ "development-tools::cargo-plugins" ]
 license = "Apache-2.0 OR MIT"
 
 [[bin]]
+name = "cargo-mobile"
+required-features = [ "cli" ]
+
+[[bin]]
 name = "cargo-apple"
 required-features = [ "cli" ]
 

--- a/src/update.rs
+++ b/src/update.rs
@@ -16,6 +16,7 @@ use std::{
 static ENABLED_FEATURES: &[&str] = &[
     #[cfg(feature = "brainium")]
     "brainium",
+    "cli",
 ];
 
 #[derive(Debug)]

--- a/src/util/git/repo.rs
+++ b/src/util/git/repo.rs
@@ -157,7 +157,7 @@ impl Repo {
                 .run_and_wait()
                 .map_err(Error::FetchFailed)?;
             self.git()
-                .command_parse("reset --hard origin/master")
+                .command_parse("reset --hard origin/dev")
                 .run_and_wait()
                 .map_err(Error::ResetFailed)?;
             self.git()

